docs/release.md

```diff
@@ -0,0 +1,53 @@
+# Releasing Gutenberg
+
+This document is a checklist for building and releasing a new version of Gutenberg. Unless you have commit access to the WordPress.org plugin repository, it's unlikely to be of much use to you. 🙂
+
+## Writing the Release Post and Changelog
+
+* Open the [https://github.com/WordPress/gutenberg/pulls?page=2&q=is%3Apr+is%3Aclosed+sort%3Aupdated-desc](recently updated PRs view), and find the PR where the last version bump occurred.
+* Read through each PR since then, to determine if it needs to be included in the Release Post and/or changelog.
+* Choose a feature or two to highlight in the release post - record an animation of them in action.
+* Save the draft post on [https://make.wordpress.org/core/](make.wordpress.org/core), for publishing after the release.
+
+## Bumping the Version
+
+* Create a PR like [https://github.com/WordPress/gutenberg/pull/2370/files](this one), bumping the version number in `gutenberg.php` and `package.json`.
```

Comment:
`package-lock.json` also needs to be bumped, see this more recent PR for reference: https://github.com/WordPress/gutenberg/pull/3479/files

Reply:
Whee!